core/src/main/java/com/linecorp/armeria/common/SystemPropertyFlagsProvider.java

```diff
@@ -438,6 +438,26 @@ public Path defaultMultipartUploadsLocation() {
         return getAndParse("defaultMultipartUploadsLocation", Paths::get);
     }
 
+    @Override
+    public Sampler<? super RequestContext> requestContextLeakDetectionSampler() {
+        final String spec = getNormalized("requestContextLeakDetectionSampler");
+        if (spec == null) {
+            return null;
+        }
+        if ("true".equals(spec) || "always".equals(spec)) {
```

Comment:
How about modifying `Samplers.of(String)` to accept `true` and `false`? Then we don't need these `if` blocks anymore.